_episodes/introduction.md

```diff
@@ -1,19 +1,115 @@
+those wishing to enhance the reusability of their data holdings." This guideline
+has subsequently been endorsed by working groups, funding bodies and
+institutions.
+
+The FAIR principles have a relatively strong focus on "machine-actionability".
+That means that the data should not only be *fair* for humans,
+but also for computers. This is relevant especially for data discovery and also
+to facilitate working with the data.
+
+![FAIR principles illustration by Scriberia](../fig/FAIRPrinciples.jpg) *This
+image was created by Scriberia for The Turing Way community and is used under a
+CC-BY licence. Source: <https://doi.org/10.5281/zenodo.3695300>*
+
+> ## FAIR is not open
+>
+> Some data cannot be openly shared for example because of privacy considerations.
+> For such situations the following guideline from the EU H2020 programme may be followed:
+>
+> *"As open as possible, as closed as necessary."*
+>
+{: .callout}
+
+## Why FAIR?
+
+The original authors of the FAIR principles had a strong focus on enhancing
+reusability of data. This ambition is embedded in a broader view on knowledge
+creation and scientific exchange. If research data are easily discoverable and
+re-usable, this lowers the barriers to repeat, verify, and build upon previous
+work. The authors also state that this vision applies not just to data, but to
+all aspects of the research process. This is visualized in the image below.
+
+![Reproducibility illustration by Scriberia](../fig/ReproducibleJourney.jpg) *This
+image was created by Scriberia for The Turing Way community and is used under a
+CC-BY licence. Source: <https://doi.org/10.5281/zenodo.3695300>*
+
+> ## What's in it for you?
+>
+> Many funders nowadays require data management plans in which the researchers
+> describe their adherence to the FAIR principles. A great step forward, but if 
+> we are going to spend the time and effort, we prefer to find some intrinsic 
+> motivation.
+>
+> Have a look at the original [Nature paper]() on FAIR data, the [GO-FAIR
+> living document](https://www.go-fair.org/fair-principles/), and the Danish
+> tutorial [howtofair.dk](https://www.howtofair.dk/why-fair/). What are the main
+> reasons for *you* to (not) embrace the FAIR principles?
+{: .discussion}
+
+## FAIRification is a process
+
+It is important to realize that the FAIR principles are not rules or standards.
+They are a set of guiding principles, but their implementation may differ per
+field or even per project. Since the principles have originally been established
+only in 2016, a comprehensive set of specific requirements for FAIR data is
+lacking for most fields.
+
+In this tutorial, we will therefore interpret "FAIRness" as a continous scale
+(FAIR vs. more FAIR) rather than a binary classification (FAIR vs. not FAIR).
+Some steps can already be taken today, whereas other steps require the
+development of standards across a scientific domain. We will aim to identify
```

Comment:
```suggestion
development of standards across a scientific domain, or even changes to the funding model of projects. We will aim to identify
```

Reply:
A lot of the problems are political rather than technical. We should mention this.